src/static/login.html

```diff
@@ -325,6 +391,20 @@
                 xhr.send();
             }
 
+            function login(user, password) {
+                var headers = {
+                    "Authorization": "Basic " + btoa(utf8(user + ":" + password))
+                };
+                send_login_request("GET", headers, false);
+            }
+
+            function converse(id, msg) {
+                var headers = {
+                    "Authorization": "Login-Reply " + id + " " + btoa(utf8(msg))
+                };
+                send_login_request("GET", headers, true);
```

Comment:
I think this is good. Stays true to the fact that it's authentication.

Can we make it 'X-Login-Reply'?